src/components/Introduction.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <h1>Introduction </h1>
+    <p> NOTE: Styling is incoming! </p>
+    <p> 
+      This dashboard has been designed as an interactive means to use the AIFRM Equity Risk Service. 
+      Below this is an executive summary of the history and current status of this report. That is followed by a quick summary
+      of the unique estimation features that were utilized and then finally the steps to use the dashboard are provided. 
+
+    </p>
+
+    <h2> Executive Summary </h2>
+    <p>
+        AIFMRM’s Equity Risk Service is a continuation of the same service previously offered by BNP Paribas
+        Securities South Africa (Pty) Ltd (BNP). This service is aimed at providing up-to-date risk measures and
+        associated statistics for the stocks and indices that are listed on the Johannesburg Stock Exchange (JSE).
+        The Equity Risk Service is based not only on the American and the British experience but also on an
+        ongoing research programme at the University of Cape Town (UCT). The estimates are based on a price
+        and volume series database sourced from the JSE Ltd, Bloomberg L.P. and INET BFA - IRESS.   
+    </p>
+
+    <h2> Unique Estimation Features </h2>
```

Comment:
I would get rid of this since we can add this later in the page (we kind of need to grab the users attention with the initial part of the page)